src/benchmark.cc

```diff
@@ -830,11 +830,13 @@ State::State(size_t max_iters, bool has_x, int x, bool has_y, int y,
 void State::PauseTiming() {
   // Add in time accumulated so far
   CHECK(running_benchmark);
```

Comment:
Sort of. 'running_benchmark' checks that the benchmark was actually started by the library. So that check would still fire if a user constructed their own 'State' object and tried to use it.